packages/components/app/styles/components/tabs.scss

```
@@ -40,7 +44,9 @@
   align-items: center;
   height: var(--token-tabs-tab-height);
   margin: 0;
-  padding: var(--token-tabs-tab-padding-vertical) var(--token-tabs-tab-padding-horizontal);
+  // padding: var(--token-tabs-tab-padding-vertical) var(--token-tabs-tab-padding-horizontal);
+  padding-left: 20px;
+  padding-right: 10px;
```

This change didn't have the effect you were looking for. The original line of CSS results in the padding on the top and bottom using the variable for `--token-tabs-tab-padding-vertical`, and the padding for the left and right using the variable for `--token-tabs-tab-padding-horizontal`. 

What you have resulted in a left padding of 20px and a right padding of 10px, which makes the content off-center. Instead what you're probably looking for is something like this:

```suggestion
  // padding: var(--token-tabs-tab-padding-vertical) var(--token-tabs-tab-padding-horizontal);
  padding: 10px 20px;
```

Here's some more information about the shorthand property for padding: https://www.w3schools.com/css/css_padding.asp